frontend/src/Settings/MediaManagement/MediaManagement.js

```diff
@@ -200,6 +200,40 @@ class MediaManagement extends Component {
                         />
                       </FormGroup>
 
+                      <FormGroup
+                        advancedSettings={advancedSettings}
+                        isAdvanced={true}
+                        size={sizes.MEDIUM}
+                      >
+                        <FormLabel>Import Using Script</FormLabel>
+
+                        <FormInputGroup
+                          type={inputTypes.CHECK}
+                          name="useScriptImport"
+                          helpText="Copy files for importing using a script, say for transcoding"
```

Comment:
Should we be a bit more formal here `Copy files for importing using a script (ex. for transcoding)`